python/sglang/srt/model_executor/forward_batch_info.py

```diff
@@ -702,7 +716,7 @@ def compute_position_torch(
     return positions.to(torch.int64), extend_start_loc
 
 
-@torch.compile(dynamic=True, backend=get_compiler_backend())
+@torch.compile(dynamic=True, backend=get_compiler_backend(), disable=_is_hpu)
```

Comment:
Can we please change this to a more common place. 
Like create an util func like need_disable() , so that in future we can have multiple scenarios covered in one util func. Anyway this part needs to be added for torch compile mode, so we can disable this part only for lazy mode, so only _is_hpu() check is not enough.

Reply:
Maybe we should address that in a follow up pr for `torch.compile`?